src/test/java/edu/indiana/dlib/amppd/repository/CollectionRepositoryTests.java

```diff
@@ -64,7 +89,6 @@ public void deleteAllBeforeTests() throws Exception {
 								jsonPath("$._embedded.collections[0].name").value( "121")) ; }
```

Comment:
I was hoping there is an easier way to call factory to create objects, i.e. not have to call setters per field. In ruby, there is elegant simple way to init a hashmap in one line, so i wondered if java has similar features. I found some in java 9 Map.of, not as clean as Ruby but at least one can do in one line https://www.baeldung.com/java-initialize-hashmap

Reply:
We cannot directly use "map.put" as in this example. This is because the subclass attributes are not visible and hence cannot be set using reflection. However I have made is concise and used setters as I did in ItemRepositoryTests. It is much simpler in the new push. Have a look.